src/esp/bindings_js/bindings_js.cpp

```diff
@@ -204,6 +203,15 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
       .function("setLocalTransform", &Sensor_setLocalTransform)
       .function("specification", &Sensor::specification);
 
+  em::class_<SensorSuite>("SensorSuite")
```

Comment:
Took a deeper look and none of the demos except the vr demo will call getSensorSuite, but there are some issues that I posted about in slack and am still investigating. 

emscripten can't use reference types to interact with js, only pointers and primitives https://stackoverflow.com/questions/55905148/emscripten-pass-stl-c-map-parameter

It seems there's a workaround to write a policy to get pointers from references, but I'm not sure if this is what we want to do here.  https://github.com/emscripten-core/emscripten/issues/3480

Can anyone provide some guidance?

Reply:
How about have JS getSensors return pointers instead of references? And then instead of binding it directly to SensorSuite::getSensors, bind it to some new C++ helper function that constructs the map of pointers.

Reply:
Thank you for the suggestion eundersander! vr demo is fixed and working